libs/tof-lib/src/lib/r4/fhir.ts

```diff
@@ -14649,7 +14649,7 @@ export class ImplementationGuideResourceComponent extends BackboneElement {
   public name?: string;
   public description?: string;
   public exampleBoolean? = false;
-  public exampleCanonical?: string;
+  public exampleCanonical?: string[];
```

Comment:
why make this an array?
according to [this](http://hl7.org/fhir/implementationguide.html) the `example[x]` field is 0..1
further-more, exampleBoolean and exampleCanonical are both options of `example[x]`, so you can't have exampleBoolean be 0..1 and exampleCanonical be 0..*

Reply:
So, I think I got myself slightly confused when I looked at the specification since meta.profile is a string array (0..*) but exampleCanonical is not.  I wasn't sure if it made sense to leave exampleCanonical as just a string when the possibility exists that there may be more than one URL set in meta.profile.  If I should just stick with the standard, I'll revert these changes and make sure that only the first URL of meta.profile is taken.  Please let me know.

Thanks!